packages/quantic/cypress/integration/facets/timeframe-facet/timeframe-facet-expectations.ts

```diff
@@ -0,0 +1,203 @@
+import {InterceptAliases} from '../../../page-objects/search';
+import {should} from '../../common-selectors';
+import {
+  BaseFacetSelector,
+  FacetWithValuesSelector,
+} from '../facet-common-selectors';
+import {
+  TimeframeFacetSelectors,
+  WithDateRangeSelector,
+} from './timeframe-facet-selectors';
+
+function validateUaEvent(
```

Comment:
Can't we use this function for all our UA validations? Even outside of the timeframe facet.

Reply:
Of course. I decided to keep the common function in this file to avoid conflicts with the upcoming refactor from this PR. But I see it would not conflict after all. Moving the function in a separate file right now.